op-interop-mon/flags/flags.go

```diff
@@ -0,0 +1,66 @@
+package flags
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/urfave/cli/v2"
+
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	opmetrics "github.com/ethereum-optimism/optimism/op-service/metrics"
+	"github.com/ethereum-optimism/optimism/op-service/oppprof"
+	oprpc "github.com/ethereum-optimism/optimism/op-service/rpc"
+)
+
+const EnvVarPrefix = "OP_INTEROP_MON"
+
+func prefixEnvVars(name string) []string {
+	return opservice.PrefixEnvVar(EnvVarPrefix, name)
+}
+
+var (
+	// Required Flags
+	L2RpcsFlag = &cli.StringSliceFlag{
+		Name:     "l2-rpcs",
+		Usage:    "The RPC URLs for the L2 chains to monitor",
+		EnvVars:  prefixEnvVars("L2_RPCS"),
+		Required: true,
+	}
```

Comment:
I wonder if we could sanity check the config against a depset loaded from the SCR somehow? 

If we had a typo or missed out an endpoint, I think we could get false positives for invalid messages.